include/dali/core/geom/vec.h

```diff
@@ -69,14 +67,15 @@ using fvec = vec<N, float>;
   using prefix##vec16 = prefix##vec<16>;
 
 DEFINE_VEC_ALIASES(i)
+DEFINE_VEC_ALIASES(i64)
```

Comment:
i32, u32 are not listed here and there is a typo above.

Reply:
I added them, with a remark that they should only be used when you _really_ need to put the number there.